File: openapi-generator/kotlin-spring/src/main/kotlin/org/openapitools/model/InlineResponse2005VehicleStats.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import org.openapitools.model.AuxInputSeries
import org.openapitools.model.EngineState
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param engineState 
 * @param auxInput2 
 * @param vehicleId ID of the vehicle.
 * @param auxInput1 
 */
data class InlineResponse2005VehicleStats (

        @get:NotNull 
        @JsonProperty("vehicleId") val vehicleId: Long,

        @JsonProperty("engineState") val engineState: List<EngineState>? = null,

        @JsonProperty("auxInput2") val auxInput2: AuxInputSeries? = null,

        @JsonProperty("auxInput1") val auxInput1: AuxInputSeries? = null
) {

}

File: openapi-generator/scala-gatling/src/gatling/scala/org/openapitools/client/model/InlineResponse2005VehicleStats.scala

package org.openapitools.client.model


case class InlineResponse2005VehicleStats (
    _engineState: Option[List[EngineState]],
    _auxInput2: Option[AuxInputSeries],
    /* ID of the vehicle. */
    _vehicleId: Long,
    _auxInput1: Option[AuxInputSeries]
)
object InlineResponse2005VehicleStats {
    def toStringBody(var_engineState: Object, var_auxInput2: Object, var_vehicleId: Object, var_auxInput1: Object) =
        s"""
        | {
        | "engineState":$var_engineState,"auxInput2":$var_auxInput2,"vehicleId":$var_vehicleId,"auxInput1":$var_auxInput1
        | }
        """.stripMargin
}
